swift_package/BrainFlowCI/BrainFlowBindings/BoardShim.swift

```diff
@@ -0,0 +1,582 @@
+     * data, for some boards we can not split EEG\EMG\... and return the same array
+     */
+    static func getPPGchannels (_ boardId: BoardIds, _ preset: BrainFlowPresets = .ANCILLARY_PRESET) throws -> [Int32] {
+        var len: Int32 = 0
+        var channels = [Int32](repeating: 0, count: 512)
+        let errorCode = get_ppg_channels (boardId.rawValue, preset.rawValue, &channels, &len)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return Array(channels[0..<Int(len)])
+    }
+
+    /**
+     * get row indices in returned by get_board_data() 2d array which contain accel
+     * data
+     */
+    static func getAccelChannels (_ boardId: BoardIds, _ preset: BrainFlowPresets = .AUXILIARY_PRESET) throws -> [Int32] {
+        var len: Int32 = 0
+        var channels = [Int32](repeating: 0, count: 512)
+        let errorCode = get_accel_channels (boardId.rawValue, preset.rawValue, &channels, &len)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return Array(channels[0..<Int(len)])
+    }
+
+    /**
+     * get row indices in returned by get_board_data() 2d array which contain analog
+     * data
+     */
+    static func getAnalogChannels (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> [Int32] {
+        var len: Int32 = 0
+        var channels = [Int32](repeating: 0, count: 512)
+        let errorCode = get_analog_channels (boardId.rawValue, preset.rawValue, &channels, &len)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return Array(channels[0..<Int(len)])
+    }
+
+    /**
+      * get row indices in returned by get_board_data() 2d array which contain gyro
+      * data
+      */
+    static func getGyroChannels (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> [Int32] {
+        var len: Int32 = 0
+        var channels = [Int32](repeating: 0, count: 512)
+        let errorCode = get_gyro_channels (boardId.rawValue, preset.rawValue, &channels, &len)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return Array(channels[0..<Int(len)])
+    }
+
+    /**
+     * get row indices in returned by get_board_data() 2d array which contain other
+     * data
+     */
+    static func getOtherChannels (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> [Int32] {
+        var len: Int32 = 0
+        var channels = [Int32](repeating: 0, count: 512)
+        let errorCode = get_other_channels (boardId.rawValue, preset.rawValue, &channels, &len)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return Array(channels[0..<Int(len)])
+    }
+
+    /// If the board is streaming or playback, then return the board ID itself. Otherwise return the master board ID.
+    static func getMasterBoardID(boardId: BoardIds, params: BrainFlowInputParams) throws -> BoardIds {
```

Comment:
make this method private or rename to getBoardId and make it non static(other bindings have getBoardId method)

Reply:
Will do.